ci-operator/config/quay/quay-tests/.config.prowgen

```diff
@@ -0,0 +1,2 @@
+private: true
```

Comment:
What does it give?

Reply:
@dmage this means allow Openshift CI to onboarding private repo, here the repo is Quay QE's test codes repo "quay-test", see https://docs.ci.openshift.org/docs/architecture/private-repositories/#openshift-priv-organization